README.md

```diff
@@ -99,149 +144,169 @@ This Dockerfile currently contains all installed C libraries and the Python wrap
+* `N_nu_rel`: Number of relativisitic species
+* `N_nu_mass`: N_nu_mass
+* `mnu`: deneutrino masssc
 * `w0`: Dark energy eqn of state parameter
 * `wa`: Dark energy eqn of state parameter, time variation
-* `H0`: Hubble's constant in km/s/Mpc.
-* `h`: Hubble's constant divided by (100 km/s/Mpc).
-* `A_s`: amplitude of the primordial PS
-* `n_s`: index of the primordial PS
+* `h`: Hubble constant in units of 100 km/s/Mpc
+* `norm_pk`: the normalization of the power spectrum, either A_s or sigma_8
+* `n_s`: the power-law index of the primordial power spectrum
+* `nz_mgrowth`: the number of redshifts where the modified growth is provided
+* `zarr_mgrowth`: the array of redshifts where the modified growth is provided
+* `dfarr_mgrowth`: the modified growth function vector provided
+* `status`: Status flag. 0 if there are no errors, nonzero otherwise.
+
+For some specific cosmologies you can also use functions **`ccl_parameters_create_flat_lcdm`**, **`ccl_parameters_create_flat_wcdm`**, **`ccl_parameters_create_flat_wacdm`**, **`ccl_parameters_create_lcdm`**, which automatically set some parameters. For more information, see file ***include/ccl_core.c***.
 
-For some specific cosmologies you can also use functions **`ccl_parameters_create_flat_lcdm`**, **`ccl_parameters_create_flat_wcdm`**, **`ccl_parameters_create_flat_wacdm`**, **`ccl_parameters_create_lcdm`**, which automatically set some parameters. For more information, see file `include/ccl_core.c`.
+The status flag `int status = 0` is passed around in almost every `CCL` function. Normally zero is returned while nonzero if there were some errors during a function call. For specific cases see documentation for **`ccl_error.c`**.
 
 ### The `ccl_cosmology` object
-For the majority of CCL's functions you need an object of type **`ccl_cosmology`**, which can be initalize by function **`ccl_cosmology_create`**
+For the majority of `CCL`'s functions you need an object of type **`ccl_cosmology`**, which can be initalize by function **`ccl_cosmology_create`**
 ```c
 ccl_cosmology * ccl_cosmology_create(ccl_parameters params, ccl_configuration config);
 ```
-Note that the function returns a pointer. Variable **`params`** of type **`ccl_parameters`** contains cosmological parameters created in previous step. Structure **`ccl_configuration`** contains information about methods for computing transfer function, matter power spectrum and mass function (for available methods see `include/ccl_config.h`). For now, you should use default configuration **`default_config`**
+Note that the function returns a pointer. Variable `params` of type **`ccl_parameters`** contains cosmological parameters created in previous step. Structure **`ccl_configuration`** contains information about methods for computing transfer function, matter power spectrum and mass function (for available methods see ***include/ccl_config.h***). For now, you should use default configuration `default_config`
```

Comment:
I would remove "For now". We do have other capabilities. I would rephrase this sentence as: "In the default configuration, CCL will use the following set-up:"